templates/federated_reporting/import.sh

```diff
@@ -124,9 +125,17 @@ log "Attaching schemas"
 for file in $dump_files; do
   if [ ! -f "${file}.failed" ]; then
     hostkey=$(basename "$file" | cut -d. -f1)
+    logfile="$WORKDIR"/outputs/"$hostkey"-schema-attach-$(date +%F-%T)-failure.log
     "$CFE_BIN_DIR"/psql -U $CFE_FR_DB_USER -d cfdb --set "ON_ERROR_STOP=1" \
                         -c "SET SCHEMA 'public'; SELECT attach_feeder_schema('$hostkey', ARRAY[$table_whitelist]);" \
```

Comment:
@nickanderson how would you integrate/configure debug logging for federated imports?

We can add
```
                        -c "SET client_min_messages TO DEBUG5" \
```

In this line and any other psql commands we run here and capture logs as I have done here to track down problems easier.z 

Maybe an augment to set the log level or just "debug" as a boolean would be easier.

Reply:
First thing that comes to mind would be a new variable, defaulted to NOTICE but overridable via augments.

Then someone could set that variable to any of the allowed values:

DEBUG5, DEBUG4, DEBUG3, DEBUG2, DEBUG1, LOG, NOTICE, WARNING, and ERROR.

Reply:
Oh, that's a shell script, not a renderd mustache file.

Reply:
my proposal here doesn't work yet... bash quoting :'(

Reply:
I tested what I have here now and think this looks good. I bailed on trying to figure out bash escaping, not worth the effort so just copy/pasted the command twice.